xkb.c

```diff
@@ -297,13 +297,29 @@ xkb_reload_keymap(void)
     }
 }
 
+void
+xkb_refresh(void)
+{
+    lua_State *L = globalconf_get_lua_State();
+
+    if (globalconf.xkb_reload_keymap)
+        xkb_reload_keymap();
+    if (globalconf.xkb_map_changed)
+          signal_object_emit(L, &global_signals, "xkb::map_changed", 0);
+    if (globalconf.xkb_group_changed)
+          signal_object_emit(L, &global_signals, "xkb::group_changed", 0);
+
+    globalconf.xkb_reload_keymap = false;
+    globalconf.xkb_map_changed = false;
+    globalconf.xkb_group_changed = false;
```

Comment:
The inner C dev in me say this should be using enum flags instead of 3 booleans. This would be faster because you could do `if (!xkb_reload_component) return;` and shorter because the reset line would be `xkb_reload_component = XKB_RELOAD_NONE;`. But as you would say, this is bikeshedding. Feel free to ignore this. Thanks for the patch.